### PR TITLE
readme corrections

### PR DESCRIPTION
**Pull request by Jaewongtongsoup**
fixed both smb and sf2t readme

### CODE REVIEW

File: Games/Retro_ML.StreetFighter2Turbo/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Training AIs to beat Street Fighter 2 Turbo fights automatically.
 
 ## Credits
```
Comment (Jaewongtongsoup) [inline Games/Retro_ML.StreetFighter2Turbo/README.md:6]:
```suggestion
## Special Thanks
```

File: Games/Retro_ML.StreetFighter2Turbo/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Training AIs to beat Street Fighter 2 Turbo fights automatically.
...
```
Comment (Tourmi) [inline Games/Retro_ML.StreetFighter2Turbo/README.md]:
```suggestion
## Special Thanks

```

File: Games/Retro_ML.StreetFighter2Turbo/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Training AIs to beat Street Fighter 2 Turbo fights automatically.
...
 
-* [GameFaq](https://gamefaqs.gamespot.com/snes/588701-street-fighter-ii-turbo/faqs/63633): Special thanks to johnlike728. His list helped to scope the range of attacks. 
+* [johnlike728](https://gamefaqs.gamespot.com/snes/588701-street-fighter-ii-turbo/faqs/63633): Special thanks to johnlike728. His list helped to scope the range of attacks.
```
Comment (Tourmi) [inline Games/Retro_ML.StreetFighter2Turbo/README.md:8]:
```suggestion
* [johnlike728](https://gamefaqs.gamespot.com/snes/588701-street-fighter-ii-turbo/faqs/63633): His list helped to scope the range of attacks.
```

**Review by Tourmi — CHANGES_REQUESTED** [review]

File: Games/Retro_ML.StreetFighter2Turbo/README.md (hunk 2 above)
Comment (Jaewongtongsoup) [reply Games/Retro_ML.StreetFighter2Turbo/README.md]: resolved

**Review by Tourmi — APPROVED** [review]

**Pull request merged**